config-linux.md

```diff
@@ -115,93 +107,59 @@ There is a limit of 5 mappings which is the Linux kernel hard limit.
 
 ## Devices
 
-`devices` is an array specifying the list of devices to be created in the container.
+`devices` is an array specifying the list of devices that the MUST be available in the container.
```

Comment:
On Wed, Jan 27, 2016 at 10:47:11AM -0800, Mrunal Patel wrote:
> > -`devices` is an array specifying the list of devices to be created in the container.
> > +`devices` is an array specifying the list of devices that the MUST be available in the container.
> 
> s/that the/that/

Fixed in 81f726e → a855883.